### PR TITLE
OSD Contribution : Add ARIA:Cell Role

### PR DESCRIPTION
**Pull request by Saran51**
# What Changed
Added ARIA Cell Role
# Why
Feature request

Todo:
- [ ] Add Semantic Version Label 
- [ ] Add tests
- [ ] Add docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,8 @@
 
 - **TimerAria** - Add `role='timer'` to indicate to assistive technologies that an element is a numerical counter listing the amount of elapsed time from a starting point or the remaining time until an end point. Add `aria-label` to provide the name of the timer. Add `aria-live` to explicitly denote a live region (Elements with the role timer have an implicit aria-live value of off). Add `aria-describedby` to indicate the idref of an element that contains additional instructions for navigating or operating this element. Add `aria-roledescription` o give the timer a more descriptive role text for screen readers to speak. Add `aria-atomic` to set whether or not the screen reader should always present the live region as a whole, even if only part of the region changes.
 
+- **CellAria** -Adds `role='cell'` aria attributes for a cell. A cell is identified as an element in a tabular container that do not contain column or row header information. “cell” are only valid within a construct that simulates a standard data table.
```
Comment (kendallgassner) [inline README.md:54]:
```suggestion
- **CellAria** - Adds `role='cell'` aria attributes for a cell. A cell is identified as an element in a tabular container that does not contain column or row header information. “cells” are only valid within a construct that simulates a standard data table.
```

File: snippets/react-snippets.code-snippets [hunk 2]
```diff
@@ -207,6 +207,20 @@
...
+        ],
+        "body": [
+            "role='cell'",
+            "aria-colspan={$1}",
+            "aria-rowspan={$2}",
+            "aria-colindex={$3}",
+            "aria-rowindex={$4}",
+            ""
+        ],
+        "description": "Adds `role='cell'` aria attributes for a cell. A cell is identified as an element in a tabular container that do not contain column or row header information. “cell” are only valid within a construct that simulates a standard data table."
```
Comment (kendallgassner) [inline snippets/react-snippets.code-snippets:222]:
```suggestion
        "description": "Adds `role='cell'` aria attributes for a cell. A cell is identified as an element in a tabular container that does not contain column or row header information. “cells” are only valid within a construct that simulates a standard data table."
```

**Review by kendallgassner — COMMENTED** [review]
Looking good!! ✨ a couple of small changes and then we cna merge !

**Comment by Saran51** [comment]
> Looking good!! ✨ a couple of small changes and then we cna merge !

I have committed your suggesstions

**Review by kendallgassner — APPROVED** [review]
⭐

**Pull request merged**